docs/CONTRIBUTING.md

```diff
@@ -357,14 +380,69 @@ You can run individual unit tests by using the following syntax:
 Or by adding `.only` after a `describe` or `it` block in the test file:
 
 ```typescript
-describe.only('Extension manifest tests', () => {
-  it('should register all commands', () => {
+describe.only("Extension manifest tests", () => {
+  it("should register all commands", () => {
     // test code
   });
 });
 ```
 
-Remember to remove `.only` after running the test and before making a PR! 
+Remember to remove `.only` after running the test and before making a PR!
+
+#### Clicktesting the VS Code extension against the sidecar
+
+1. Clone the [`ide-sidecar` repo](https://github.com/confluentinc/ide-sidecar) and check out the
+   branch/commit you want to test against. (Also ensure any sidecar
+   [prerequisites](https://github.com/confluentinc/ide-sidecar?tab=readme-ov-file#prerequisites-for-development)
+   are installed.)
+
+2. Within the cloned `ide-sidecar` directory (make sure to install the
+   [prerequisites](https://github.com/confluentinc/ide-sidecar?tab=readme-ov-file#prerequisites-for-development)),
+   and make sure to run the following `make` command to build the native executable.
```

Comment:
No need to mention the sidecar prereqs again in this item since it's linked in the item above.
```suggestion
2. Within the cloned `ide-sidecar` directory, run the following `make` command to build the native executable.
```